File: apps/hotel-booking-service-admin/src/api/payment/Payment.ts
import { Booking } from "../booking/Booking";

export type Payment = {
  amount: number | null;
  booking?: Booking | null;
  createdAt: Date;
  description: string | null;
  id: string;
  name: string | null;
  paymentDate: Date | null;
  status?: "Option1" | null;
  updatedAt: Date;
};

File: apps/dotnet-service/src/APIs/Payment/Dtos/Payment.cs
using DotnetService.Core.Enums;

namespace DotnetService.APIs.Dtos;

public class Payment
{
    public double? Amount { get; set; }

    public string? Booking { get; set; }

    public DateTime CreatedAt { get; set; }

    public string? Description { get; set; }

    public string Id { get; set; }

    public string? Name { get; set; }

    public DateTime? PaymentDate { get; set; }

    public StatusEnum? Status { get; set; }

    public DateTime UpdatedAt { get; set; }
}
